packages/core/src/index.ts

```diff
@@ -158,20 +160,30 @@ export function buildSendMail<T>(options: BuildSendMailOptions<T>) {
             templateName: templateName || derivedTemplateName,
             previewName,
             mailOptions,
+            listId,
           }),
         });
 
         if (hookResponse.status === 200) {
           const {
             message: { id: messageId },
+            memberId,
           } = await hookResponse.json();
+
+          console.log(listId);
+
           const stringHtml = mailOptions.html?.toString();
           if (stringHtml) {
             mailOptions.html = instrumentHtml({
               html: stringHtml,
               messageId: messageId,
               apiUrl: MAILING_API_KEY,
             });
+
+            stringHtml.replace(
```

Comment:
@psugihara what's the best way to verify this works manually?

Reply:
i think this could work:
- add a template with an unsubscribe link
- use the PreviewSender to manually send yourself one